### ISSUE TITLE
Any chance later gaming joysticks will be emulated?

### ISSUE THREAD

**Issue by BridgeHeadland** [joncampbell123/dosbox-x#4576]
### Question

According to the wiki, DOSBox-X does not emulate later game controllers, such as the MS SideWinder. I don't think this joystick article has been updated for a while, if I'm not mistaken, but work is being done to get DOSBox-X to emulate newer controllers, including controllers used for Windows, which DOSBox-X can run ? Can it be done?
As far as I know, a newer DOS PC will even support XBox 360 controllers (also the slim controller, I'd think), since Windows XP, which DOSBox-X (as you know) can run, has support for the last mentioned controller. I don't know about any newer controllers, as a DOS PC will support.

### Have you checked that no similar question(s) exist?

- [X] I have searched and didn't find any similar question.

### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

**Comment by rderooy** [comment]
No, the DOSBox-X wiki article is up-to-date. No work is being done to enhance the joystick support.

And also, no, USB or Bluetooth game controllers will not natively work on a retro DOS PC.

**Comment by BridgeHeadland** [comment]
@rderooy This means that you cannot run ant kind of DOS as a host operating system on a PC with a USB port, nor with support for Bluetooth?

**Comment by rderooy** [comment]
Who says so? I did not. Perhaps you should re-read what I wrote.

**Comment by Torinde** [comment]
USB and Bluetooth can be used under later Win9x/XP versions and many of the PCs running those also can run DOS. But per my knowledge there are very few DOS drivers for USB (mostly mouse?) and none for Bluetooth.

Side note: a topic I think both of you have expertise to collaborate on - and I would be eager to see the result - is to make Wiki guides for installing NT-based Windows #3538
- https://dosbox-x.com/wiki/Guide%3AMicrosoft-Windows-in-DOSBox‐X
- Windows NT 3.1
- Windows NT 3.5
- Windows NT 3.51
- Windows NT 4.0
- Windows 2000
- Windows XP
- Windows 2003 (incl. R2 SP2)
- ReactOS

Having a rderooy's guide for NT is kind of officializing it's support...

- adding to [the wiki](https://dosbox-x.com/wiki/Guide%3ADOS-Installation-in-DOSBox‐X) a Guide for DR-DOS, the other major branch of DOS would be interesting as well

**Comment by rderooy** [comment]
I don't think creating guides for Windows versions beyond WinME is wise. The last thing I think we want is people actually trying to do that. DOSBox-X is not meant to run those versions, especially in the current state.

If you want to run NT versions of Windows, just use something like virtualbox, vmware or kvm.
